src/contexts/SolanaWalletContext.tsx

```diff
@@ -20,13 +21,40 @@ import {
   BloctoWalletAdapter,
   BraveWalletAdapter,
 } from "@solana/wallet-adapter-wallets";
-import { FC, useMemo } from "react";
 import { CLUSTER, SOLANA_HOST } from "../utils/consts";
 
+declare global {
+  interface Navigator {
+    brave: {
+      isBrave(): Promise<boolean>;
+    };
+  }
+  interface Global {
```

Comment:
If you omit declaring the Global interface here, does typescript gives you an error?

I'm almost sure you can delete this and everything should work fine, window.navigator it's already defined globally for typescript